docs/source/howto/setup_codes.rst

```diff
@@ -0,0 +1,185 @@
+.. _how-to:setup_code:
+
+*******************
+How to setup a code
+*******************
+
+Once you have at least one computer configured (see :ref:`how-to:setup_computer`), you can configure the codes.
+In AiiDA, for full reproducibility of each calculation, we store each code in the database, and attach to each calculation a given code. This has the further advantage to make very easy to query for all calculations that were run with a given code (for instance because I am looking for phonon calculations, or because I discovered that a specific version had a bug and I want to rerun the calculations).
```

Comment:
```suggestion
In AiiDA, for full reproducibility of each calculation, we store each code in the database, and attach to each calculation a given code.
This has the further advantage to make it very easy to query for all calculations that were run with a given code (for instance because I am looking for phonon calculations, or because I discovered that a specific version had a bug and I want to rerun the calculations).
```

Reply:
closing as changed after leo's merges